### PR TITLE
Support removing users from groups.

### PR DESCRIPTION
**Pull request by mattspitz**
Added a 'remove' button next to the 'edit' button that removes a user
from a group via modal.

Users must be owners to remove members from groups. Users may not remove
themselves (use "leave group" instead).

### CODE REVIEW

**Comment by mattspitz** [comment]
https://www.dropbox.com/s/6e0gt3qvwiae6nw/GrouperRemoveFromGroupDemo.mov?dl=0

File: grouper/fe/handlers.py [hunk 1]
```diff
@@ -841,6 +842,40 @@ def post(self, group_id=None, name=None):
...
 
 
+class GroupRemove(GrouperHandler):
+    def post(self, group_id=None, name=None):
+        group = Group.get(self.session, group_id, name)
+        if not group:
+            return self.notfound()
+
+        members = group.my_members()
+        if self.current_user.my_role(members) != "owner":
```
Comment (tmildorf) [inline grouper/fe/handlers.py]: Managers see remove buttons for members, but when they attempt to remove a member, their efforts fail here.

Comment (mattspitz) [inline grouper/fe/handlers.py]: Good catch. My mistake. Fixed!

File: grouper/fe/handlers.py [hunk 2]
```diff
@@ -869,6 +870,40 @@ def post(self, group_id=None, name=None):
...
+
+        removed_member = get_user_or_group(self.session, member_name, user_or_group=member_type)
+
+        if self.current_user == removed_member:
+            return self.send_error(
+                status_code=400,
+                reason="Can't remove yourself. Leave group instead."
+            )
+
+        group.revoke_member(self.current_user, removed_member, "Removed by owner.")
```
Comment (tmildorf) [inline grouper/fe/handlers.py]: Might be a manager rather than owner.

Comment (mattspitz) [inline grouper/fe/handlers.py]: Sorted!

**Pull request merged**